src/Aspire.Hosting.Azure/AzurePublishingContext.cs

```diff
@@ -163,14 +163,14 @@ private async Task WriteAzureArtifactsOutputAsync(IPublishingStep step, Distribu
             // Map parameters from existing resources
             if (resource.TryGetLastAnnotation<ExistingAzureResourceAnnotation>(out var existingAnnotation))
             {
-                Visit(existingAnnotation.ResourceGroup, MapParameter);
-                Visit(existingAnnotation.Name, MapParameter);
+                await VisitAsync(existingAnnotation.ResourceGroup, MapParameterAsync, cancellationToken).ConfigureAwait(false);
+                await VisitAsync(existingAnnotation.Name, MapParameterAsync, cancellationToken).ConfigureAwait(false);
             }
 
             // Map parameters for the resource itself
             foreach (var parameter in resource.Parameters)
             {
-                Visit(parameter.Value, MapParameter);
+                await VisitAsync(parameter.Value, MapParameterAsync, cancellationToken).ConfigureAwait(false);
```

Comment:
Remove uses of parameter.Value in this file.

Reply:
Hmm revert this. That's not what I meant. I mean ParameterResource.Value should be replaced with ParameterResource.GetValueAsync

Reply:
Undo that change as well